docs/_includes/projects/properties/accessorssubclassing/my-element.js

```diff
@@ -1,16 +1,39 @@
-import SuperElement from './super-element.js';
+import { LitElement, html } from 'lit-element';
 
-class MyElement extends SuperElement {  
-  static get properties() { return {}; };
-  
-  get prop1() {
-    return this._prop1;
+export class MyElement extends LitElement {
+  static get properties() { return {
+    prop1: { type: Number },
+  };}
+
+  set prop1(newVal) {
+    let oldVal = this._prop1;
+    this._prop1 = Math.floor(newVal);
+    this.requestUpdate('prop1', oldVal);
   }
 
-  set prop1(value) {
-    console.log('MyElement custom setter');
-    this._prop1 = Math.floor(value/10)*2;
+  get prop1() { return this._prop1; }
+
+  constructor() {
+    super();
+    this._prop1 = 0;
+  }
+
+  render() {
+    return html`<p>prop1: ${this.prop1}</p>
+
+    <button @click="${this.getNewVal}">change</button>
+  `;
+  }
+
+  updated(changedProperties) {
```

Comment:
Do we need these methods? I think it clouds the critical parts of the example and we could just have a getter and setter without any math or `update()`.